crates/bevy_remote/src/schemas/json_schema.rs

```diff
@@ -538,6 +616,32 @@ mod tests {
             "a"
           ]
         });
-        assert_eq!(schema_as_value, value);
+        assert_normalized_values(schema_as_value, value);
+    }
+
+    fn assert_normalized_values(one: Value, two: Value) {
+        let mut one = one.clone();
+        normalize_json(&mut one);
+        let mut two = two.clone();
+        normalize_json(&mut two);
+        assert_eq!(one, two);
```

Comment:
Nit: Do we have to clone here? We're taking `one` and `two` by value, so we could probably just mutate them directly.
```suggestion
    fn assert_normalized_values(mut one: Value, mut two: Value) {
        normalize_json(&mut one);
        normalize_json(&mut two);
        assert_eq!(one, two);
```